src/components/views/messages/MPollBody.tsx

```diff
@@ -51,6 +52,7 @@ export default class MPollBody extends React.Component<IBodyProps, IState> {
         this.state = {
             selected: null,
             pollRelations: this.fetchPollRelations(),
+            seenEventIds: [],
```

Comment:
this probably doesn't need to be state, but doesn't hurt in this case. Class-private properties are a bit more suited for unrendered bits of information.

Reply:
Thanks, good point.  Done in 4156ba11e8